Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -680,8 +794,12 @@ public function setCode($code)
             $this[$index] = new Token($token);
         }
 
+        $transformers = Transformers::create();
+        $transformers->transform($this);
```

Comment:
1.4 should have been tagged. i really don't think we should accept anything else into it